src/rustdoc_schema.graphql

```diff
@@ -499,12 +499,18 @@ type Method implements Item & FunctionLike {
 A specific attribute applied to an Item.
 """
 type Attribute {
-  """
-  The textual representation of the attribute.
+  as_string: String!
+  is_inner: Boolean!
+  content: AttributeValue!
+}
 
-  For example: "#[repr(C)]"
-  """
-  value: String!
+type AttributeValue {
+  base: String!  # the path, e.g. "cfg_attr" or "must_use"
+  as_string: String!  # the entire value represented as a string
+  assigned_expression: String  # the string form of the assigned expression, or null if there wasn't one
```

Comment:
I'm inclined to suggest `assigned_value` here instead of `assigned_expression`: it would be consistent with `full_value / raw_value` above, and `expression` is kind of a "scary" word.

In general, whenever it doesn't hurt something else, it's good to use less-complex language as much as possible because it makes onboarding new users faster and easier. Naming simple ideas with complex names keeps people away from learning and using them.